resources/galera_configuration.rb

```diff
@@ -174,4 +176,8 @@ def build_galera_options
     end
     galera_options
   end
+
+  def default_libgalera_smm_path
```

Comment:
We prefer to append the cookbook name to the methods so we don't have any potential conflicts.
```suggestion
  def mariadb_libgalera_smm_path
```